android/app/build.gradle

```diff
@@ -48,13 +50,37 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
+    signingConfigs {
+        release {
+            storeFile file(System.getenv("HOME") + "/keystores/allthenews.keystore")
+            storePassword System.getenv("BITRISEIO_ANDROID_KEYSTORE_PASSWORD")
+            keyAlias System.getenv("BITRISEIO_ANDROID_KEYSTORE_ALIAS")
+            keyPassword System.getenv("BITRISEIO_ANDROID_KEYSTORE_PRIVATE_KEY_PASSWORD")
+        }
+    }
+
     buildTypes {
         release {
-            // TODO: Add your own signing config for the release build.
-            // Signing with the debug keys for now, so `flutter run --release` works.
-            signingConfig signingConfigs.debug
+            signingConfig signingConfigs.release.storeFile.exists() ? signingConfigs.release : signingConfigs.debug
         }
     }
+
+    flavorDimensions "flavor-type"
```

Comment:
zastanawiam się czy to potrzebne skoro jest jeden dimension 🤔

Reply:
zgodnie z dokumentacją 

> All flavors must belong to a named flavor dimension, which is a group of product flavors. You must assign all flavors to a flavor dimension; otherwise, you will get the build error shown below. If a given module specifies only one flavor dimension, the Android Gradle plugin automatically assigns all of the module's flavors to that dimension.

https://developer.android.com/studio/build/build-variants#product-flavors

Leci error jak nie podamy żadnego